app/webpacker/css/admin/connected_apps.scss

```diff
@@ -68,3 +68,19 @@
     }
   }
 }
+
+solid-permissioning {
+  @import url("https://fonts.googleapis.com/css2?family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap");
```

Comment:
Is it able to inherit the default OFN font instead of loading a new one?

Reply:
I need to experiment with that. The first version was overriding some styles but we should be in control now.